nodebuilder/share/module.go

```diff
@@ -38,6 +37,7 @@ func ConstructModule(tp node.Type, cfg *Config, options ...fx.Option) fx.Option
 		return fx.Module(
 			"share",
 			baseComponents,
+			fx.Invoke(share.EnsureEmptySquareExists),
```

Comment:
does this happen somewhere implicitly for nodes running eds store?

Reply:
Yes, inside the start function for edsStore (ensureEmptyCarExists)

Reply:
kk